<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add</class>
 <widget class="QDialog" name="add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QLineEdit" name="le_nom">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 1px solid;
border-color: #9B0000;
border-radius: 10px;
}</string>
   </property>
   <property name="placeholderText">
    <string>Nom..</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="le_date">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateEdit{
border: 1px solid;
border-color: #9B0000;
border-radius: 9px;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_id">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 1px solid;
border-color: #9B0000;
border-radius: 10px;
}</string>
   </property>
   <property name="placeholderText">
    <string>ID..</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_domaine">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 1px solid;
border-color: #9B0000;
border-radius: 10px;
}</string>
   </property>
   <property name="placeholderText">
    <string>Domaine..</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_ajouter">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>340</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(158,59,59);
font: 75 9pt &quot;Arial&quot;;
border: 2px solid rgb(158,59,59);
border-radius: 15px;
}

QPushButton:hover {
  color: #fff; 
  background-color: #a63b3b; 
  border-color: #a63b3b; 
}

QPushButton:pressed {
  color: #fff; 
  background-color: #802c2c; 
  border-color: #802c2c; 
}</string>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>361</width>
     <height>331</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressources.qrc">:/new/prefix1/gfx/3516835.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="captcha">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>113</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>le_nom</zorder>
  <zorder>le_date</zorder>
  <zorder>le_id</zorder>
  <zorder>le_domaine</zorder>
  <zorder>b_ajouter</zorder>
  <zorder>captcha</zorder>
  <zorder>label_8</zorder>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
